ivy_tests/test_ivy/helpers/hypothesis_helpers/array_helpers.py

```diff
@@ -50,7 +50,8 @@ def array_bools(
     [True]
 
     """
-    size = size if isinstance(size, int) else draw(size)
+    if not isinstance(size, int):
+        size = draw(size)
```

Comment:
`size` will be assigned to itself if `isinstance(size, int)`

So instead of assigning it to itself, we just check whether it is an int or not, if int, leave it as it is, if it not an int then `size = draw(size)`

Reply:
It kind of doesn't do anything except that it skips a useless command of `size = size`

```
if isinstance(size, int):
    size = size
else:
    size = draw(size)
```

Reply:
Yeah, makes sense! I will go ahead and merge this PR